modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/util/JavaSourceUtil.java

```diff
@@ -37,6 +41,40 @@
  */
 public class JavaSourceUtil extends SourceUtil {
 
+	public static String addImport(
+			String content, String fileName, String... newImportNames)
+		throws IOException, ParseException {
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		List<String> importNames = javaClass.getImportNames();
```

Comment:
What happens if the Java file has no imports? I know that for the upgrade check, this is impossible. But, we are adding this `addImport()` method to `JavaSourceUtil`, which could be used by normal SF checks down the line.

Reply:
But you use the imports to know where to put it. So if you don't have imports, where exactly would the new import names go?

Reply:
What would happen here: https://github.com/kevhlee/liferay-portal/pull/72/files#diff-e63471b5fe762f8c4b9bf12b5cc884d8e27409c0e8f4e0327945b3cfd64b98c6R63?

Reply:
It is up to you how you want to deal with the case where there are no imports. If you plan to keep `addImport()` in `JavaSourceUtil`, then you need to handle the zero imports case. If plan to not keep it in there, you could just move the method into the check where you have the guarantee that there are imports.

Reply:
Well noticed, Tomorrow morning I will adjust those logic. Any suggestions on how to add on this empty scenario?

Reply:
It would be slightly tricky. I would try to use the location the package name. So maybe using a regex to capture package declarations and putting the imports under there.